addons/fullscreen/addon.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "Enhanced fullscreen",
+  "description": "Adds several tweaks and customization options to the project player's fullscreen mode, such as syncing with your browser's fullscreen mode (F11), darkening the background behind the project player, and hiding the green flag toolbar.",
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://github.com/lisa-wolfgang"
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "If you hide the toolbar and don't have the fullscreen sync option enabled, you'll need to use the Esc key on your keyboard to exit the project player's fullscreen mode.",
```

Comment:
Maybe we can shorten this to `You can always use the Esc key to exit fullscreen mode.`?

Reply:
Whoops, the notice below 😂